### PR TITLE
[WFLY-16720/16721] Remove undefine for legacy security realm in helloworld-mutual-ssl[-secured] quickstart

### PR DESCRIPTION
**Pull request by cam-rod**
- https://issues.redhat.com/browse/WFLY-16720
- https://issues.redhat.com/browse/WFLY-16721

Depends on [migration to Jakarta EE 10](https://issues.redhat.com/browse/WFLY-16730), or usage on pre-Big Bang server.

### CODE REVIEW

**Review by fjuma — APPROVED** [review]
Thanks @cam-rod!

File: helloworld-mutual-ssl-secured/restore-configuration.cli [hunk 1]
```diff
@@ -4,8 +4,7 @@
 batch
 
 # Revert the changes in the undertow subsystem
-/subsystem=undertow/server=default-server/https-listener=https:undefine-attribute(name=ssl-context)
-/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=security-realm,value=ApplicationRealm)
+/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context,value=applicationSSC)
```
Comment (jbliznak) [inline helloworld-mutual-ssl-secured/restore-configuration.cli:7]: `applicationSSC` is some default ssl-context that we are restoring to this value?

Comment (jbliznak) [reply helloworld-mutual-ssl-secured/restore-configuration.cli:7]: ok, nevermind, I found the answer here https://docs.wildfly.org/26/WildFly_Elytron_Security.html#out-of-the-box-configuration :wink:

Comment (cam-rod) [reply helloworld-mutual-ssl-secured/restore-configuration.cli:7]: actually didn't know about that section, TIL 😁

**Comment by fjuma** [comment]
@emmartins When you get a chance, it would be good to merge this one since I have further changes coming for these quickstarts. Thanks.

**Comment by bstansberry** [comment]
@cam-rod @fjuma I'm closing this as it appears to be out of date now; i.e. none of the changed lines exist in main. Thanks!

**Pull request closed without merging**